webpack.demo.js

```diff
@@ -0,0 +1,54 @@
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: {
+    'angular2': [
+      'zone.js',
+      'reflect-metadata',
```

Comment:
What about es6-shim? I guess we should include it in the demo so it works in all the supported browsers. Here again I would just stick it in a script tag